Assets/InventoryApp.cs

```diff
@@ -0,0 +1,112 @@
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro; // Include the TextMesh Pro namespace
+using System;
+
+public class InventoryApp : MonoBehaviour
+{
+    public Button obtainPotionButton;
+    public Button usePotionButton;  // Button to use a potion, assign in Unity Editor
+    public TMP_Text potionCountText;
+    public TMP_Text cooldownText;
+
+    private int potionCount;
+    private float cooldownTime = 20f;
+    private float nextUseTime;
+
+    void Start()
+    {
+        potionCount = PlayerPrefs.GetInt("HealthPotions", 0);
```

Comment:
Better introduce a `const` variable/field for this and use it in both `GetInt` and `SetInt`